src/_data/resources.json

```diff
@@ -143,6 +148,11 @@
 			"description": "Building an elegant, functional website requires more than just knowing how to code. In Adaptive Web Design, Second Edition, you’ll learn how to use progressive enhancement to build websites that won’t break, work anywhere, are accessible by anyone—on any device, and are designed to work well into the future.",
 			"additional": "Aaron Gustafson",
 			"url": "http://a.co/d/7aVobuX"
+		},
+				{
+			"title": "Agile Accessibility Handbook",
+			"description": "Free ebook: A Practical Guide to Accessible Software Development at Scale": "Dylan Barrell",
```

Comment:
It looks like the nested quotes are breaking the build. Let's try this instead:


`"description": "Free ebook: A Practical Guide to Accessible Software Development at Scale, by Dylan Barrell.",`